packages/types/src/types/requests/datatrak-web-server/UserRequest.ts

```diff
@@ -12,7 +12,12 @@ export type Params = Record<string, never>;
 export interface ResBody {
   id?: string;
   userName?: string;
+  firstName?: string;
+  lastName?: string;
   email?: string;
+  employer?: string;
+  position?: string;
+  mobileNumber?: string | null;
```

Comment:
Should these properties be hard-coded, or would it be wise to derive them from my new, shared `UserAccountDetails` type?

Reply:
It occurs to me that `UserAccountDetails` exists in `datatrak-web`, not `datatrak-web-server`

Reply:
I think this is ok for now 👍